scripts/batchacc.go

```diff
@@ -0,0 +1,116 @@
+// This tool categorizes a set of test names (camel-cased strings) into reduced set of prefixes
```

Comment:
Would put this in the `tools` dir rather than scripts, just because it's Go program.

Reply:
SGTM